netd.yaml

```diff
@@ -158,6 +162,11 @@ spec:
       terminationGracePeriodSeconds: 0
       nodeSelector:
         beta.kubernetes.io/kube-netd-ready: "true"
+      tolerations:
```

Comment:
The tolerations will match all the nodes, is this your purpose? 
https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/